proxy/proxy.go

```diff
@@ -230,12 +249,26 @@ func copyHeader(to, from http.Header) {
 	}
 }
 
+func copyHeaderExcluding(to, from http.Header, excludeHeaders map[string]bool) {
+	for k, v := range from {
+		if len(excludeHeaders) == 0 || !excludeHeaders[k] {
```

Comment:
Why 2 checks here?
This should be enough, not?:

    !excludeHeaders[k] {

The more Goish way would be:

    if _, ok := excludeHeaders[k]; !ok {

This would check only that there is a key within the map.

Reply:
Thanks for the tip on the Go(ish) way of making that if-statement. I've change the code to reflect that. I will test the request with `ConNection` and get back to you on that.